reference/call-home.md

```diff
@@ -0,0 +1,110 @@
+# Call-home metrics 
+
+ In the interest of gaining perspective on storage usage trends, Mayastor collects some data metrics from the deployed Mayastor instances. Installing Mayastor using Helm chart (v2.0.0 onwards) deploys a Kubernetes deployment called `obs-callhome`on to the cluster. A Kubernetes pod from this deployment collects the data points from your Mayastor cluster. The collected data is anonymous and is encrypted at rest. 
```

Comment:
```suggestion
 In the interest of gaining perspective on storage usage trends, DataCore Software Inc. collects some data metrics from the deployed Mayastor instances. Installing Mayastor using the Helm chart (v2.0.0 onwards) deploys a Kubernetes deployment called `obs-callhome` on to the cluster. A Kubernetes pod from this deployment collects the data points from your Mayastor cluster. The collected data is _anonymous_ and is encrypted at rest. 
```

Reply:
I think that this is in the correct spirit but the wrong approach.  DataCore isn't the collector/requestor of the information.  That is the OpenEBS project itself, or at least its maintainers.  DataCore is _donating storage and processing resource_  in support of this.

I suggest going with "Mayastor collects" here (i.e. as per the original edit), and then adding a section on "Where is the data stored", wherein DataCore can be named as the provider of the resources for that.  We should also list where geographically the data will be stored - i.e. US (Texas data centre) and wherever else we may maintain replicas/DR

Reply:
Wondering if we can/should mention company's name in an opensource document? cc  @GlennBullingham @avishnu

Reply:
I think that's fine, as it's both truthful and already immediately apparent that DataCore is a major contributor / maintainer of the project.